src/deepsparse/transformers/pipelines/text_classification.py

```diff
@@ -132,20 +139,31 @@ class TextClassificationPipeline(TransformersPipeline):
         If a list of lengths is provided, then for each length, a model and
         tokenizer will be compiled capable of handling that sequence length
         (also known as a bucket). Default is 128
-    :param return_all_scores: if True, instead of returning the prediction as the
+    :param top_k: number of labels with the highest score to return. Default is 1
+    :param return_all_scores: [DEPRECATED] set top_k to desired value instead.
```

Comment:
How do we go about the deprecated argument? Can't we remove it in such a way, that the change is invisible to the user?

Reply:
we want to keep it in for backwards compatibility with any existing flows for at least a release or two. transformers is doing the same with this specific argument